scripts/src/pullrequest/prepare_pr_comment.py

```diff
@@ -38,138 +43,245 @@ def get_look_at_job_output_comment():
-"""
+    msg = verifier_error_msg
+    msg = append_to(msg, get_look_at_job_output_comment())
     return msg
 
 
 def prepare_community_comment():
-    msg = f"{get_community_review_message()}\n\n"
+    msg = f"{get_community_review_message()}"
     if os.path.exists("./pr/errors"):
         errors = open("./pr/errors").read()
-        msg += "However, please note that one or more errors were found while building and verifying your pull request:\n\n"
-        msg += f"{errors}\n\n"
+        msg = append_to(
+            msg,
+            "However, **please note** that one or more errors were found while building and verifying your pull request:",
+        )
+        msg = append_to(msg, f"{errors}")
     return msg
 
 
 def prepare_generic_fail_comment():
     msg = ""
     if os.path.exists("./pr/errors"):
         errors = open("./pr/errors").read()
-        msg += "One or more errors were found while building and verifying your pull request:\n\n"
-        msg += f"{errors}\n\n"
+        msg = append_to(
+            msg,
+            "One or more errors were found while building and verifying your pull request:",
+        )
+        msg = append_to(msg, f"{errors}")
     else:
-        msg += "An unspecified error has occured while building and verifying your pull request.\n\n"
+        msg = append_to(
+            msg,
+            "An unspecified error has occured while building and verifying your pull request.",
+        )
     return msg
 
 
 def prepare_oc_install_fail_comment():
-    msg = "Unfortunately the certification process failed to install OpenShift Client and could not complete.\n\n"
-    msg += "This problem will be addressed by maintainers and no further action is required from the submitter at this time.\n\n"
-    return msg
+    return " ".join(
+        [
+            "Unfortunately the certification process failed to install OpenShift Clientand could not complete.",
+            "This problem will be addressed by maintainers and no further action is required from the submitter at this time.",
+        ]
+    )
 
 
-def get_comment_header(issue_number):
-    msg = (
-        f"Thank you for submitting PR #{issue_number} for Helm Chart Certification!\n\n"
+def append_to(msg, new_content, use_horizontal_divider=False):
+    """Appends new_content to the msg.
+
+    This utility function helps simplify the building of our PR comment
+    template. msg and new_content are joined with either a newline separator, or
+    a horizontal line (if the keyword argument is provided).
+
+    It should be used in cases where the caller needs to join semi-related
+    ideas. Callers should instead use the join string method in cases where the
+    msg being constructed is a part of the same 'idea', or 'paragraph'.
+
+    Developer Note: The use of the strip method is a developer optimization.
+    It's intended to allow us to use a multi-line string but work entirely at
+    the same indentation level in source code.
```

Comment:
I removed the developer note and instead added a comment explicitly where `strip()` is called.

The gist here is that multi-line strings preserve whitespace. I'm adding the multiline string instantiator and terminator (e.g. `"""`) on new lines, which would artificially add newlines in the string itself. This is just to keep the source code easy to read, and so I strip them out using `strip()` so they don't end up in our result.

If it's too unclear, and you prefer me to just skip using strip and change the source, I can do that.

Reply:
Thanks for the explanation. It's clear now.

I was mostly thrown off by the comment regarding indentation level in the source code. We do need to kind of mess up the indentation when working with multi-line string (as the content of this multi-line string cannot be indented with the rest of the code.) I interpreted your initial comment as trying to solve this issue, which confused me.

I understand now that the intention is to keep the same indentation level "inside" the multi-line string content i.e. avoid doing:

```
    return f"""{msg}
{divider_string}
{new_content}"""
```

I'm okay with using `strip()` here, and we should leave your comment.

Just note that this will also strip out any leading whitespace in the `msg` string. I don't think that's an issue.